stages/org.osbuild.users

```diff
@@ -134,7 +141,8 @@ def add_ssh_key(root, user, key):
         os.chown(ssh_dir, int(uid), int(gid))
 
     with open(authorized_keys, "a", encoding="utf8") as f:
-        f.write(f"{key}\n")
+        for key in keys:
+            f.write(f"{key}\n")
```

Comment:
This can be simplified to `"\n".join(keys) + "\n"`.

Reply:
Will apply :)

Reply:
Done.